### PR TITLE
FBFW-76 - Create Territory, Labels and Unit Position for England

### PR DESCRIPTION
**Pull request by GCroom**
## Test Link:
https://6231d1586e134f6cdee73aa3--flamboyant-franklin-5232af.netlify.app/

## Territory Location Figma:
https://www.figma.com/file/JnpO52R22kG3lB2wexs9rB/webDiplomacy?node-id=3746%3A34196

## Label and Icon Location Figma:
https://www.figma.com/file/JnpO52R22kG3lB2wexs9rB/webDiplomacy?node-id=0%3A1

image

### CODE REVIEW

File: beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx [hunk 1]
```diff
@@ -229,6 +259,16 @@ const WDBoardMap: React.FC = function (): React.ReactElement {
...
+      <defs>
+        <pattern
+          id="sea-texture"
+          patternUnits="userSpaceOnUse"
+          width="1546"
+          height="1384"
+        >
+          <image href={WaterTexture} x="0" y="0" width="1966" height="1615" />
+        </pattern>
+      </defs>
```
Comment (bryancodazen) [inline beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx:271]: I think you should be able to add this to the existing `defs` container that starts on line 246.

File: beta-src/src/enums/Territory.ts [hunk 2]
```diff
@@ -5,6 +5,11 @@ enum Territory {
...
   CLYDE,
   EDINBURGH,
+  ENGLISH_CHANNEL,
+  IRISH_SEA,
+  MIDDLE_ATLANTIC,
+  NORTH_ATLANTIC,
+  NORTH_SEA,
   FINLAND,
   GASCONY,
   GREECE,
```
Comment (bryancodazen) [inline beta-src/src/enums/Territory.ts:15]: I think this file's territories should be in alpha order as well.

**Review by nebevets — APPROVED** [review]
LGTM...make bryan's changes

**Review by sariacodazen — APPROVED** [review]
LGTM once those previously mentioned touch ups are done :D

File: beta-src/package.json [hunk 3]
```diff
@@ -2,7 +2,6 @@
   "name": "beta-src",
   "version": "0.1.0",
   "private": true,
-  "homepage": "beta",
```
Comment (bryancodazen) [inline beta-src/package.json:5]: We should add this back.

**Review by bryancodazen — CHANGES_REQUESTED** [review]
One minor change.

**Review by bryancodazen — APPROVED** [review]
LGTM

**Pull request merged**